src/lib.rs

```diff
@@ -14,8 +14,8 @@
 
 use actix_rt::net::UnixStream;
 use actix_service::Service;
-use actix_tls::connect::{Connect, ConnectError, Connection};
-use awc::http::Uri;
+use actix_tls::connect::{ConnectError, ConnectInfo, Connection};
+use actix_web::http::Uri;
```

Comment:
https://actix.rs/actix-web/awc/http/struct.Uri.html

Reply:
Got it, ok, thanks for pointing it out